istioctl/pkg/cli/option.go

```diff
@@ -77,6 +83,14 @@ func (r *RootFlags) Namespace() string {
 	return *r.namespace
 }
 
+func (r *RootFlags) KubeClientTimeout() time.Duration {
+	d, err := time.ParseDuration(*r.kubeTimeout)
+	if err != nil {
+		d = 15 * time.Second
```

Comment:
Are we sure we want a silent failure? I think a log message should be the minimum

Reply:
I had opted to swallow the error and fall back on defaults so we didn't change the end user experience _at all_, but I think you're right, I'll add an error log.